package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uid2/uid2-sdk",
-  "version": "4.0.0",
```

Comment:
I think going to 4.0.0 in the previous PR was unnecessary - there are no breaking changes.